Cargo.toml

```diff
@@ -13,21 +13,20 @@ members = [
 resolver = "1"
 
 [workspace.dependencies]
-cosmian_crypto_core = "9"
-async-trait = "0.1.51"
-base64 = "0.21.0"
-cloudproof_cover_crypt = "12.0"
+cosmian_crypto_core = "9.1.0"
+async-trait = "0.1.73"
+base64 = "0.21.3"
```

Comment:
Dependencies should be listed in the alphabetic order.

Reply:
indeed, thanks,
done in https://github.com/Cosmian/cloudproof_rust/pull/43/commits/86eabae65617e47e52edd44b11cc7872ca7b2330